rate_limiter/src/lib.rs

```diff
@@ -95,13 +95,27 @@ pub struct TokenBucket {
 
     // Internal state descriptors.
     budget: u64,
-    last_update: u64,
+    last_update: Instant,
 
     // Fields used for pre-processing optimizations.
     processed_capacity: u64,
     processed_refill_time: u64,
 }
 
+impl Default for TokenBucket {
```

Comment:
The default implementation for TokenBucket is not used anywhere. Please remove it in a different commit before the commit that replaces chrono and time with the custom implementation from fc_util.